src/utils/EventEmitter.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2022 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import mitt, {Emitter, EventType, Handler, WildcardHandler} from 'mitt';
+
+export class EventEmitter<Events extends Record<EventType, unknown>> {
```

Comment:
Why do we need this wrapper and cannot use mitt directly?

Reply:
to have a bit of control over the API and possible to instrument it in the future. E.g., in Puppeteer we have `removeAllListeners` implemented in the wrapper. But happy to remove the wrapper as it is strictly speaking is not needed.

Reply:
Agree, it's makes API more explicit having this wrapper